Backend/controllers/ResumeController.js

```diff
@@ -0,0 +1,62 @@
+import Resume from "../models/Resume.js";
+
+/**
+ * Gets all Resumes from database server and return as a promise.
+ * 
+ * @returns Resolved promise containing all resumes objects.
+ * @returns Rejected promise containing reject causing error.
+ */
+export const getAllResumes = async () => {
+    try {
+        const resumes = await Resume.find({}, {__v: 0});
+        return Promise.resolve({ resumes });
+    } catch (error) {
+        return Promise.reject({ status: 500, ...error });
+    }
+}
+
+export const getResume = async (id) => {
+    try {
+        const resume = await Resume.findById(id, { __v: 0 });
+
+        if (!resume) return Promise.reject({ status: 400 });
+
+        return Promise.resolve({ resume });
+    } catch (error) {
+        return Promise.reject({ status: 500, ...error });
+    }
+}
+
+export const addResume = async (resumeDetails) => {
+    try {
+        const resume = await Resume.create(resumeDetails);
+        return Promise.resolve({ resume });
+    } catch (error) {
+        return Promise.reject({ status: 500, ...error });
+    }
+}
+
+export const deleteResume = async (id) => {
+    try {
+        const deletedResume = await Resume.findByIdAndDelete(id);
+
+        if (!deletedResume) return Promise.reject({ status: 400 });
+
+        return Promise.resolve(deletedResume);
```

Comment:
Maintain consistency by sending an `object`, with fetched item inside, in resolved promise as done in line `12`, `24` and `33`.
Also do the same in line no. `58`.

Reply:
fixed in commit 342bea0